front/src/modules/trainschedule/components/ImportTrainSchedule/generatePathfindingPayload.ts

```diff
@@ -1,119 +0,0 @@
-import { compact, isEmpty } from 'lodash';
-
-import type { Step, TrainScheduleWithPathRef } from 'applications/operationalStudies/types';
-import type { LightRollingStock } from 'common/api/osrdEditoastApi';
-import rollingstockOpenData2OSRD from 'modules/trainschedule/components/ImportTrainSchedule/rollingstock_opendata2osrd.json';
-import { isFirstOrLastElement } from 'utils/array';
```

Comment:
Maybe you can remove this function `isFirstOrLastElement`

Reply:
Good catch, done.